scripts/copy-res.js

```diff
@@ -45,8 +65,32 @@ function next(i, err) {
     const source = ent[0];
     const dest = ent[1];
     const opts = ent[2] || {};
+    let cpx = undefined;
 
-    const cpx = new Cpx.Cpx(source, dest);
+    if (opts.languages) {
+        const sourceFiles = generateFileArray(source);
+        let Sourcelanguages = {};
+        if (!fs.existsSync(dest)) {
+            fs.mkdirSync(dest);
+        }
+        sourceFiles.forEach(file => {
+            const fileContents = fs.readFileSync(source + file).toString();
+            Sourcelanguages[file] = JSON.parse(fileContents);
+        });
+        sourceFiles.forEach(file => {
+            if (!fs.existsSync(dest + file)) {
+                let o = Object.assign({}, Sourcelanguages[file]);
+                fs.writeFileSync(dest + file, JSON.stringify(o, null, 4));
+            } else {
+                const fileContents = fs.readFileSync(dest + file).toString();
+                let o = Object.assign(JSON.parse(fileContents), Sourcelanguages[file]);
+                fs.writeFileSync(dest + file, JSON.stringify(o, null, 4));
```

Comment:
I think what's happening here is that you're generating all of the language files for every language file in the sources (ie. each file will be generated n times).